src/vm/eventpipeprotocolhelper.cpp

```diff
@@ -61,6 +66,7 @@ const EventPipeCollectTracingCommandPayload* EventPipeCollectTracingCommandPaylo
     uint32_t bufferLen = BufferSize;
     if (!TryParseCircularBufferSize(pBufferCursor, bufferLen, payload->circularBufferSizeInMB) ||
         !TryParseSerializationFormat(pBufferCursor, bufferLen, payload->serializationFormat) ||
+        !TryParseRundownRequested(pBufferCursor, bufferLen, payload->rundownRequested) ||
```

Comment:
After Preview7 snap we are done with breaking changes. This will need to be done as a new command (ie CollectTracing2). CollectTracing should have the behavior it always had, rundown is enabled always.

Reply:
Should be fairly easy to re-use the payload class/parsing logic for the most part and simply provide a default value of `true` when being populated by the original `CollectTracing` command.

Reply:
Thanks Noah - I just updated the PR to add a new command and tested against previous versions of tools and confirmed they still work.